### DISCUSSION TITLE
Mountain Duck file modification date

### DISCUSSION THREAD

**Discussion by cedrickp32** [iterate-ch/cyberduck#14600] [Q&A]
When uploading a file, the modification date is not preserved. This make the server have a "newer" file even if it's not. I'm using a backup software with Google Drive mounted, and it cause a lot of problem since it tried to copy the server files on the local drive because the modification date is wrong.

CyberDuck has a setting in Settings > Transfers > Timestamps > Preserve modification date.

**Comment by dkocher** [comment]
Preserving modification dates is the default with Mountain Duck. Cannot be reproduced copying files in Finder.app or Windows Explorer.
